libraries/botbuilder-core/src/telemetryConstants.ts

```diff
@@ -15,4 +15,5 @@ export class TelemetryConstants {
     public readonly textProperty: string = 'text';
     public readonly speakProperty: string = 'speak';
     public readonly userIdProperty: string = 'userId';
+    public readonly attachmentsProperty: string = 'attachments';
```

Comment:
These class properties should have been static like in [C#](https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Builder/TelemetryConstants.cs). From a cursory glance on the class' references developers shouldn't need to instantiate this class to access these values.